src/Components/Server/src/Circuits/CircuitHost.cs

```diff
@@ -810,7 +810,7 @@ public static void BeginInvokeDotNetFailed(ILogger logger, string callId, string
         }
 
 #pragma warning disable SYSLIB1025
-        [LoggerMessage(219, LogLevel.Error, "Location change to '{URI}' in circuit '{CircuitId}' failed.", EventName = "LocationChangeFailed")]
+        [LoggerMessage(219, LogLevel.Error, "Location change to '{URI}' in circuit '{CircuitId}' failed.", EventName = "LocationChangeFailedInCircuit")]
```

Comment:
What event name do you get if you remove `EventName` entirely? I've never specified the Event Name so I figured the default would be that is uses the method name. If that's the case then it would be even simpler.

Reply:
Hello @Eilon. Thank you for the question. Generally if the event name attribute is not provided when defining a `LoggerMessage`, the logging system will generate a default event name based on the `LoggerMessage `type and the position of the `LoggerMessage `definition in the source code.

The default event name is generated by concatenating the namespace, class name, and method name of the `LoggerMessage `definition, separated by dots in between. For now, I have provided the event name attribute to maintain the syntax uniformity in the code. If required I think it can be removed.